src/ostorlab/agent/mixins/agent_persist_mixin.py

```diff
@@ -225,3 +225,39 @@ def add_ip_network(
         else:
             member_value = ip_range.exploded
         return self.set_add(key, member_value)
+
+    def ip_network_exists(
```

Comment:
Consider employing this method within the `add_ip_network` function to eliminate redundancy.

Reply:
The add_ip_network role is to add it if its not there, what i want is just to check if it exists. for consistency i prefer doing this separately.